Makefile

```diff
@@ -58,8 +63,12 @@ ifeq ($(NOBULLSEYE), 0)
 	$(MAKE_WITH_RETRY) EXTRA_DOCKER_TARGETS=$(notdir $@) BLDENV=bullseye -f Makefile.work bullseye
 endif
 ifeq ($(NOBOOKWORM), 0)
-	$(MAKE_WITH_RETRY) BLDENV=bookworm -f Makefile.work $@
+	$(MAKE_WITH_RETRY) EXTRA_DOCKER_TARGETS=$(notdir $@) BLDENV=bookworm -f Makefile.work $@
```

Comment:
Can you remove `EXTRA_DOCKER_TARGETS` for Bookworm? Otherwise, that may change current master branch behavior when building a specific target.

Reply:
Done.